Mathlib/Topology/Sets/OpenCover.lean

```diff
@@ -0,0 +1,54 @@
+/-
+Copyright (c) 2025 David Loeffler. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: David Loeffler
+-/
+
+import Mathlib.Topology.Sets.Opens
+
+/-!
+# Open covers
+
+We define `IsOpenCover` as a predicate on indexed families of open sets in a topological space `X`,
+asserting that their union is `X`. This is an example of a declaration whose name is actually
+longer than its content; but giving it a name serves as a way of standardizing API.
+-/
+
+open Set Topology
+
+namespace TopologicalSpace
+
```

Comment:
I think we actually want to place this in the `Topology` namespace. That's how, for example, [Topology.IsInducing](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Topology/Defs/Induced.html#Topology.IsInducing) are done. There was a Zulip thread somewhere about switch namespaced things from `TopologicalSpace` to `Topology` (but keeping the name of the class as `TopoogicalSpace`), but I can't seem to find it now.

Reply:
`Opens` is in the `TopologicalSpace` namespace, though, and I think we should be consistent with that.

(To me, it somehow feels like `IsInducing` is saying something _about the topology itself_, while `IsOpenCover` is saying something about some other object which _depends on_ the topology.)

Reply:
(I'm going to close this sub-conversation as "resolved" with the code as it stands, but if there's a future PR that moves `Opens` etc to the `Topology` namespace, I certainly have no objection to this getting moved as well.)